db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/FromBaseTable.java

```diff
@@ -3410,7 +3408,11 @@ private String getClassName(String niceIndexName) throws StandardException {
         if(niceIndexName!=null){
             cName = "IndexScan["+niceIndexName+"]";
         }else{
-            cName = "TableScan["+getPrettyTableName()+"]";
+            cName = "TableScan["+getPrettyTableName();
+            if(pastTxId >= 0){
+                cName += "timeTravelTx(" + pastTxId + ")";
```

Comment:
Please add a space before timeTravelTx.
Also, you could simply write "as of" instead, to echo the query, but that's unimportant.

Reply:
I think the general term for this feature is time travel. Therefore I would like to keep using this term for consistency.

Reply:
I will fix the extra-space.